File: 백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        while (true) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            int[] input = new int[3];
            input[0] = Integer.parseInt(st.nextToken());
            input[1] = Integer.parseInt(st.nextToken());
            input[2] = Integer.parseInt(st.nextToken());
            if (input[0] == 0 && input[1] == 0 && input[2] == 0) {
                break;
            }

            Arrays.sort(input);

            int max = input[2];
            int mid = input[1];
            int min = input[0];

            if (max * max == (mid * mid) + (min * min)) {
                sb.append("right").append("\n");
            } else {
                sb.append("wrong").append("\n");
            }
        }

        System.out.print(sb);
    }
}

File: Python/백준/Bronze/4153. 직각삼각형/직각삼각형.py
while True:
    sides = list(map(int, input().split()))
    if sides[0] == 0 and sides[1] == 0 and sides[2] == 0: break

    max = 0
    mid = 0
    min = 0
    for side in sides:
        if side > max:
            min = mid
            mid = max
            max = side
        elif side > mid:
            min = mid
            mid = side
        elif side > min:
            min = side

    result = "wrong"
    if max * max == (mid * mid + min * min):
        result = "right"

    print(result)